src/typings.ts

```diff
@@ -93,4 +104,10 @@ export type ReminderProposal = ProposalExtended & {
   suggestion: (ProposalExtended & { corrections: Proposal[] }) | null;
 };
 
-export type Proposals = Correction | Suggestion;
+export type Reports = Report & GodfathersDecisions;
+
+export interface ReportExtended extends Reports {
+  suggestion: Suggestion;
+}
```

Comment:
Y'a pas de raison que Report prenne un S, c'est un typage c'est un singulier.
```suggestion
export interface ReportExtended extends Report, GodfathersDecisions {
  suggestion: Suggestion;
}
```